aep/general/0217/aep.md.j2

```diff
@@ -1,5 +1,96 @@
 # Unreachable resources
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/20) for more
-information.
+Occasionally, a user may ask for a list of resources, and some set of resources
+in the list are temporarily unavailable. For example, a user may ask to list
+resources across multiple parent locations, but one of those locations is
+temporarily unreachable. In this situation, it is still desirable to provide
+the user with all the available resources, while indicating that something is
+missing.
+
+## Guidance
+
+If a method to retrieve data is capable of partially failing due to one or more
+resources being temporarily unreachable, the response message **must** include
+a field to indicate this:
+
+{% tab proto %}
+
+```proto
+message ListBooksResponse {
+  // The books matching the request.
+  repeated Book books = 1;
+
+  // The next page token, if there are more books matching the
+  // request.
+  string next_page_token = 2;
+
+  // Unreachable resources.
+  repeated string unreachable = 3;
+}
+```
+
+{% tab oas %}
+
+**Note:** OAS example note yet written.
+
+{% endtabs %}
+
+- The field **must** be a repeated string/string array, and **should** be named
+  `unreachable`.
+- The field **must** be set to the paths of the resources which are the cause
+  of the issue, such as the parent or individual resources that could not be
+  reached. The objects listed as unreachable **may** be _parents_ (or higher
+  ancestors) rather than the individual resources being requested. For example,
+  if a location is unreachable, the location is listed.
+  - The response **must not** provide any other information about the issue,
+    such as error details or codes. To discover what the underlying issue is,
+    users **should** send a more specific request.
```

Comment:
Maybe say users need to send a more specific request? Should/Must seems reserved for API design rules, not user behaviors.

Reply:
It's actually a bit wrong, IMO, to provide any guidance about users rather than about the API - so rephrased accordingly (still a **should**, because there might be cases where an API can't practically provide more information).